packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -57,7 +57,7 @@ export interface OptimizeCiemssOperationState extends BaseState {
 	interventionPolicyId: string; // Used to determine if we need to reset the InterventionPolicyGroupForm.
 	interventionPolicyGroups: InterventionPolicyGroupForm[];
 	// Constraints:
-	constraintGroups: ConstraintGroup[];
+	constraintGroups: Criterion[];
```

Comment:
Are we going all the way and rename the field as well? e.g. it still says "state.constraintGroups"

Reply:
It would be great to do so. But the PR is quite big already, and I thought stopping here will be good for now.
I want to keep doing the cleaning, maybe in another PR soo.